packages/flutter/lib/src/material/shadows.dart

```diff
@@ -6,55 +6,70 @@ import 'dart:ui' show Color, Offset;
 
 import 'package:flutter/painting.dart';
 
-const Map<int, List<BoxShadow>> shadows = const <int, List<BoxShadow>>{
+// Based on http://www.google.com/design/spec/what-is-material/elevation-shadows.html
+
+const Color _kKeyUmbraOpacity = const Color(0x33000000); // CSS alpha = 0.2
+const Color _kKeyPenumbraOpacity = const Color(0x24000000); // CSS alpha = 0.14
+const Color _kAmbientShadowOpacity = const Color(0x1F000000); // CSS alpha = 0.12
+
+const Map<int, List<BoxShadow>> elevationToShadow = const <int, List<BoxShadow>>{
```

Comment:
The elevations don't have names so an enum might not improve this.

It might be helpful to define some elevation constants. Like Elevations.kPopupMenu, Elevations.kFloatingActionButtonRaised, etc.